lib/ansible/modules/net_tools/cloudflare_dns.py

```diff
@@ -254,62 +285,62 @@
                 weight: 5,
             }
         id:
-            description: the record id
+            description: The record ID.
             returned: success
             type: str
             sample: f9efb0549e96abcb750de63b38c9576e
         locked:
-            description: No documentation available
+            description: No documentation available.
```

Comment:
I don't understand the "No documentation available" description. Does that mean we do not know what the `locked` field means for Cloudflare?

Reply:
I don't know, I didn't write this module and it's not my goal to fix documentation (semantic or correctness) issues. I simply don't know much about Cloudflare.